src/localization/localize.js

```diff
@@ -40,8 +40,10 @@ for (var i = 0; i < supportedLangs.length; i++) {
 var strings = new LocalizedStrings(catalog);
 
 var findLang =
-  supportedLangs.find(sLang => window.navigator.language === sLang) ||
-  supportedLangs.find(sLang => window.navigator.language.startsWith(sLang));
+  supportedLangs.find(
+    sLang => window.navigator.language === sLang ||
+    window.navigator.language.startsWith(sLang)
+  );
```

Comment:
I think this change is not correct, but the earlier one isn't quite either. I think what you want to do is that if you have an exact match, use it. But if there are no exact matches, then use any where the first two characters match. Then fallback to English.
In this version, if we supported zh-cn and zh-hk and the browser is set to zn-hk, it would return 'zh-cn'.
In the earlier version (which is also not quite right), if we supported zh-cn and zh-hk and the browser was set to zh, it would return 'en'.
Perhaps logic like:
```
  if supportedLangs.find(slang => window.navigator.language === sLang) {
    // Use an exact match if possible
    strings.setLanguage(sLang);
  } else {
    // Use any string whose language code matches, otherwise default to 'en'
    const match = supportedLangs.find(slang => window.navigator.language.substring(0,2) === sLang.substring(0,2))
    strings.setLanguage(match || 'en');
  }
```